src/containers/tutor-profile/main-info/MainInfo.js

```diff
@@ -0,0 +1,129 @@
+import { useContext } from 'react'
+import { useSelector } from 'react-redux'
+import { useTranslation } from 'react-i18next'
+
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import IconButton from '@mui/material/IconButton'
+import EditOutlinedIcon from '@mui/icons-material/EditOutlined'
+import ContentCopyRoundedIcon from '@mui/icons-material/ContentCopyRounded'
+import SchoolIcon from '@mui/icons-material/School'
+import DoneIcon from '@mui/icons-material/Done'
+import MailOutlineIcon from '@mui/icons-material/MailOutline'
+
+import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
+import AppChipList from '~/components/app-chips-list/AppChipList'
+import IconWithTextList from '~/components/icon-with-text-list/IconWithTextList'
+
+import useBreakpoints from '~/hooks/use-breakpoints'
+import { SnackBarContext } from '~/context/snackbar-context'
+import img from '~/assets/img/tutor-profile-page/avatar.png'
+import { accountInfo, subjectChips, doneItems } from '~/containers/tutor-profile/main-info/MainInfo.constants'
+import { styles } from '~/containers/tutor-profile/main-info/MainInfo.styles'
+import { snackbarVariants } from '~/constants'
+
+const MainInfo = () => {
+  const { t } = useTranslation()
+  const { isDesktop, isMobile } = useBreakpoints()
+  const { userRole } = useSelector((state) => state.appMain)
+  const { setAlert } = useContext(SnackBarContext)
+
+  const copyProfileLink = () => {
+    navigator.clipboard.writeText(window.location.href)
+    setAlert({
+      severity: snackbarVariants.success,
+      message: 'tutorProfilePage.mainInfo.copyProfileLink',
+      duration: 2000
+    })
+  }
+
+  const editOrCopyIcon =
+    userRole === 'student' ? (
```

Comment:
if its another tutor? you shold check is this my profile or not

Reply:
fixed